test/34-firewall.spec.ts

```diff
@@ -40,6 +41,9 @@ describe('Host Firewall', function () {
 			},
 		} as Docker.Image);
 
+		await db.initialized;
+		await db.models('app').del();
```

Comment:
does this affect other test suites which depend on this table ?

Reply:
I don't think it would, because we don't want database entry persistence between test suites anyway, and this call only deletes the current rows in `app` table. 

However the fact that we have to do this manually is pretty scuffed. With mocha v8 global hooks, we can reset the database to default values before each root `describe` block. I will be (and am kinda in the process of) looking into this .